js/plugins/checks/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@genkit-ai/checks",
+  "description": "Google Checks AI Safety plugins for classifying the safety of text against Checks AI safety policies.",
+  "keywords": [
+    "genkit",
+    "genkit-plugin",
+    "google cloud",
+    "google ai",
+    "ai",
+    "genai",
+    "generative-ai",
+    "checks",
+    "google checks",
+    "guardrails"
+  ],
+  "version": "0.9.0-dev.2",
```

Comment:
This needs to be updated to match the rest. Please make sure it matches at the time that you check this PR in (currently on `0.9.0-rc.3` but may change by merge time).

Reply:
Done